tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -1798,7 +1798,9 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 			pvc, err := client.CoreV1().PersistentVolumeClaims(namespace).Get(ctx, pvcName, metav1.GetOptions{})
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 			pv := getPvFromClaim(client, namespace, pvcName)
-			verifyBidirectionalReferenceOfPVandPVC(ctx, client, pvc, pv, fcdID)
+			pvc_volume := pvc.Spec.VolumeName
+			volumeID := strings.ReplaceAll(pvc_volume, "static-pv-", "")
```

Comment:
can you add a comment here on why pv name will be `volumeID`?

Reply:
added